def_Courant_Number.py

```diff
@@ -0,0 +1,7 @@
+def_Courant_Number(time_steps, space_steps, wave_speed)
```

Comment:
This (`def_`) is a typo, so obviously can’t work.  This is why tests are important.  You should never merge code that isn’t tested.

Reply:
No docstring